dist/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <title>Weather App</title>
+<script defer src="index.js"></script></head>
+
+<body>
+    <div id="container">
+        <div class="modal" id="myModal"></div>
+        <header>
+            <div class="searchContainer">
+                <h2>Weather App</h2>
+                <form action="submit" class="formsection">
+                    <input type="text" class="searchBar" placeholder="Enter City Name" required id="cityname">
+                    <button type="button" id="search" data-bs-toggle="modal" data-bs-target="#myModal">Search</button>
+                </form>
+                <div id="weather-card">
+             
+                </div>
+             </div>        
+        </header>
+    
+        <main class="main-content" id="main-content">
+            
+        </main>
+            
+        <footer>
+            <div class="footerContainer">
+              <p>Created by Kidist and Riyaz</p>
+            </div>
+        </footer>
+           
+    </div>
+    <script src="./index.js"></script>
```

Comment:
Webpack automatically generates this link so there is no need of including it. In addition, it will also avoid duplications so kindly remove it

Reply:
Done as advsied